ModelBundle/EventListener/GeneratePathListener.php

```diff
@@ -5,25 +5,16 @@
 use Doctrine\ODM\MongoDB\Event\LifecycleEventArgs;
 use Doctrine\ODM\MongoDB\Event\PostFlushEventArgs;
 use OpenOrchestra\ModelInterface\Model\NodeInterface;
-use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerAware;
 
 /**
  * Class GeneratePathListener
  */
-class GeneratePathListener
+class GeneratePathListener extends ContainerAware
```

Comment:
It will give a redirection loop : GeneratePathListener => NodeRepository => Doctrine => DoctrineListener => GeneratePathListener